README.md

```diff
@@ -1,5 +1,15 @@
 # boats.py
-The official discord.boats API wrapper for Python
+The official [discord.boats](https://discord.boats) API wrapper for Python
+
+# Installation
+## Install via pip (Recommended)
+```pip install boats.py```
+## Install from source
+```
+git clone https://github.com/DiscordBoats/boats.py
+cd boats.py
+pip install -r requirements.txt
```

Comment:
Why not use the setup.py file?

Reply:
I normally use the requirements.txt but since we do have a setup file then Sure!